jcli/src/jcli_app/vote/tally/decryption_tally.rs

```diff
@@ -9,7 +9,7 @@ use structopt::StructOpt;
 /// The outputs are provided as hex-encoded byte sequences.
 #[derive(StructOpt)]
 #[structopt(rename_all = "kebab-case")]
```

Comment:
Is this `StructOpt` impl now unnecessary?

Reply:
It is necessary for the composition of the commands otherwise it won't compile. Or I misunderstand want you meant...

Reply:
My bad, I thought `kebab-case` concerns the command names themselves, but this is for options/subcommands.